src/i18n/resources/en.js

```diff
@@ -418,6 +418,7 @@ The FRA team
     cancel: 'Cancel',
     changeStatusTextPlaceholder: 'Add an optional message',
     doNotNotifyUsers: "Don't notify users",
+    notifySelf: 'Send copy to self',
```

Comment:
Not sure, sounds weird
Maybe:

```suggestion
notifySelf: 'Send a copy to myself',
```

Reply:
Would we want to requests translations again to keep it consistent? if so, it would take some time. We are still waiting for the French translation.

Reply:
I think it's fine to change or correct the English one if there is a typo?
All is fine, you can resolve this comment also without changing